hail/Makefile

```diff
@@ -372,3 +374,11 @@ update-hail-repl: NAMESPACE ?= default
 update-hail-repl: wheel
 	kubectl -n $(NAMESPACE) cp $(WHEEL) $$(kubectl get pods -n $(NAMESPACE) -l app=hail-repl | tail -n +2 | awk '{print $$1}'):.
 	kubectl -n $(NAMESPACE) exec -it $$(kubectl get pods -n $(NAMESPACE) -l app=hail-repl | tail -n +2 | awk '{print $$1}') -- pip3 install -U hail-$(HAIL_PIP_VERSION)-py3-none-any.whl
+
+.PHONY: configure-gcloud
+configure-gcloud:
+	gcloud config set project $(PROJECT)
+	gcloud container clusters get-credentials vdc --zone=$(ZONE)
+	docker_prefix=$(DOCKER_PREFIX)
+	docker_registry=${DOCKER_PREFIX%%/*}
```

Comment:
These should be upper case?  In either case, the definition and use of DOCKER_PREFIX should agree.

Reply:
Changed to the uppercase!